service/history/api/recordworkflowtaskstarted/api.go

```diff
@@ -408,3 +417,69 @@ func CreateRecordWorkflowTaskStartedResponse(
 
 	return response, nil
 }
+
+func CreateRecordWorkflowTaskStartedResponseWithRawHistory(
```

Comment:
so the logic here is the same as `CreateRecordWorkflowTaskStartedResponse`, only the value type is different? 

I wonder if we should have only one copy of the logic for determining the values and then a conversion func between those two types.